<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Model_Main</class>
 <widget class="QMainWindow" name="Model_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TAPARAT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="CreateFolder">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Create New Local&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Image Folder&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Cr. Folder</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;PibotoLt&quot;;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRot">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate 180</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_WB">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>204</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>25</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>WB</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DRC">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>DRC</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>161</width>
      <height>138</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
        border: 1px solid gray;
        border-radius: 5px;
        margin-top: 0.5em;
    }

QGroupBox::title {
subcontrol-origin: margin;
padding:-2 3px;
}</string>
    </property>
    <property name="title">
     <string>Exposure</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>118</x>
       <y>58</y>
       <width>25</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ISO</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>98</y>
       <width>25</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>SS</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_SS">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>92</y>
       <width>83</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_ISO">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>52</y>
       <width>83</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_auto">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>AUTO</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TIMER</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDelay">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>30000</number>
    </property>
    <property name="singleStep">
     <number>50</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineFileName">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>124</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75  8pt &quot;PibotoLt&quot;;</string>
    </property>
    <property name="text">
     <string>img_001</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShotButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Make .py file&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;on photo servers&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Shot</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LinePath">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>124</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;PibotoLt&quot;;</string>
    </property>
    <property name="text">
     <string>/mnt/nfs/</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Preview">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineDate">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;PibotoLt&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>419</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
